### PR TITLE
Add farm

### PR DESCRIPTION
**Pull request by ndmaxar**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ndmaxar** [comment]
@oblute  and @rluria14 agree to be maintainers?

File: recipes/farm/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - setuptools
+    - wheel
```
Comment (dopplershift) [inline recipes/farm/meta.yaml:23]: Kinda silly to have this as a run-time dependency, but that's what upstream does...

File: recipes/farm/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e7c7626ce80a4e532e0db3902c96630509c390e4f90a1a4f3402c496d565a79e
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [win]
```
Comment (dopplershift) [inline recipes/farm/meta.yaml:15]: This package looks like it's pure Python? Any reason not to make this `noarch: python`? We can ignore any failing non-linux builds due to missing dependencies.

File: recipes/farm/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  imports:
+    - farm
+    - farm.conversion
+    - farm.data_handler
+    - farm.evaluation
+    - farm.modeling
+    - farm.visual
+    - farm.visual.ascii
+  requires:
+    - pytest
```
Comment (dopplershift) [inline recipes/farm/meta.yaml:47]: FYI, this only installs `pytest`. If you actually intend to have it run the test suite, you need to do that manually with a `commands` section under `test`.

File: recipes/farm/meta.yaml (hunk 2 above)
Comment (ndmaxar) [reply recipes/farm/meta.yaml:15]: No specific reason. Just rusty when it comes to building packages.

**Review by dopplershift — COMMENTED** [review]
Looks good. @rluria14 can you confirm you are willing to be a maintainer for this recipe?

**Comment by ndmaxar** [comment]
@dopplershift anything else I should do?

**Review by dopplershift — APPROVED** [review]

**Pull request merged**